.github/workflows/bencher.yml

```diff
@@ -90,10 +90,10 @@ jobs:
           ninja
           # Microbenchmarks
           ./tests.sh -VV -L benchmark
-          # End to end performance tests
-          ./tests.sh -VV -L perf -C perf
-          # Convert microbenchmark output to bencher json
+          # End to end performance tests with aggregation
```

Comment:
I don't know if the comments add a lot of meaning that the script names don't already carry.